doc/support.rst

```diff
@@ -1,20 +1,26 @@
+.. _support-index:
+
 Get support
 ===========
 
-For questions or to report problems with Sphinx, join the `sphinx-users`_
-mailing list on Google Groups, come to the ``#sphinx-doc`` channel on
-`libera.chat`_, or open an issue at the tracker_.
+For questions or to report problems with Sphinx:
+
+- open a topic at the `Github discussions`_ page,
```

Comment:
I'd also add something about "Please verify that your question does not exist on StackOverflow" etc because issues are sometimes duplicated.

Reply:
Done